playbooks/common/openshift-cluster/evaluate_groups.yml

```diff
@@ -33,11 +33,11 @@
       groups: oo_masters_to_config
       ansible_ssh_user: "{{ g_ssh_user | default(omit) }}"
       ansible_sudo: "{{ g_sudo | default(omit) }}"
-    with_items: groups[g_masters_group] | default([])
+    with_items: "{{ g_master_hosts | default([]) }}"
 
-  # Use g_new_nodes_group if it exists otherwise g_nodes_group
+  # Use g_new_node_hosts if it exists otherwise g_node_hosts
   - set_fact:
-      g_nodes_to_config: "{{ g_new_nodes_group | default(g_nodes_group | default([])) }}"
+      g_node_to_config: "{{ g_new_node_hosts | default(g_node_hosts | default([])) }}"
```

Comment:
g_node_to_config should probably be g_nodes_to_config or g_node_hosts_to_config.

The current naming seems to imply a single host instead of multiple.